features/overscroll-behavior.yml

```diff
@@ -0,0 +1,29 @@
+name: overscroll-behavior
+description: The `overscroll-behavior` CSS property allows disabling default scrolling behaviors when the edges of a scrolling area are reached.
+spec: https://drafts.csswg.org/css-overscroll-1/
+caniuse: css-overscroll-behavior
+group: scrolling
+# BCD has incorrect Edge support data. This can be removed when that is resolved.
```

Comment:
If we keep that in, we should link to the BCD bug. Our future selves will likely only read this and act on it a long time after the bug has been resolved.

Reply:
A big +1 to this. Any time we can do `# TODO: <action> when <url> resolves` is a winner to me.

I've sent https://github.com/mdn/browser-compat-data/pull/24598 to do the actual fix in BCD.